router/server/shard.go

```diff
@@ -144,7 +144,7 @@ func (srv *ShardServer) Receive() (pgproto3.BackendMessage, error) {
 		Uint("single-shard", spqrlog.GetPointer(srv)).
 		Type("message-type", msg).
 		Str("txstatus", srv.TxStatus().String()).
-		Str("shard", srv.Datashards()[0].ID()).
+		Strs("shards", srv.DatashardIds()).
```

Comment:
```suggestion
		Strs("shards", srv.DatashardIds()[0]).
```
Зачем убирать индекс?

Reply:
thats because its now list of strings, not just one?

Reply:
panic: runtime error: index out of range